porcupine/_state.py

```diff
@@ -62,6 +70,12 @@ def get_parsed_args() -> Any:
     return _parsed_args
 
 
+def get_paned_window() -> ttk.Panedwindow:
+    if _pw is None:
+        raise RuntimeError("Porcupine is not running")
+    return _pw
```

Comment:
Maybe a slightly longer name than `_pw`? On the other hand, every function that uses `_pw` makes it obvious that it is the paned window.